orangecontrib/timeseries/widgets/owtabletotimeseries.py

```diff
@@ -26,100 +57,255 @@ class Inputs:
     class Outputs:
         time_series = Output("Time series", Timeseries)
 
+    class Error(widget.OWWidget.Error):
+        invalid_time = widget.Msg("Invalid time")
+        year_out_of_range = widget.Msg("{}.")
+
+    class Warning(widget.OWWidget.Warning):
+        nan_times = widget.Msg("Rows with missing values of '{}' are skipped")
+
     want_main_area = False
     resizing_enabled = False
 
-    # Old settings that can't be migrated, but can be supported
-    selected_attr = settings.Setting('')
-    radio_sequential = settings.Setting(2)
-
-    settingsHandler = DomainContextHandler()
-    implied_sequence = settings.ContextSetting(0)
-    order = settings.ContextSetting(None)
-    autocommit = settings.Setting(True)
+    implied_sequence = Setting(0)
+    steps = Setting(1)
+    unit = Setting(next(iter(StepOptions)))
+    include_extra_part = Setting(True)
+    start_date = Setting((2022, 11, 1))
```

Comment:
To commemorate the day that code was written.

Since you appear to have no respect for such historic events, I changed it to 2000-1-1, but I'll revenge by using some another date I learned today.